controls.py

```diff
@@ -294,9 +303,18 @@ def __init__(self, text, pos = Position(0, 0), font = None, size = 20):
 		self.top = pos.top
 		self.width = super().get_rect().width
 		self.height = super().get_rect().height
+		
+		self.fontPath = font
+		self.fontSize = size
+		self.text = text
+		
+	def setText(self, text):
+		self.text = text
+		self.textSurface = pygame.font.Font(self.fontPath, self.fontSize).render(text, 1, pygame.Color("black"))
 	
 	def draw(self, surface):
-		surface.blit(self, (self.left, self.top))
+		self.setText(self.text)
```

Comment:
I don't think this line is necessary so I'll probably remove it in a future commit.